ecosystem/sep-direct-payment.md

```diff
@@ -0,0 +1,318 @@
+- Sending Anchor: The business offering outbound payment services.  Takes fiat in from the sending client, and has a business relationship with the receiving anchor.
+- Receiving Anchor: The business offering inbound payment processing. Deposits fiat in the receiving clients bank account, and has a business relationship with the sending anchor.
+- Receiving Client: The owner of the destination bank account.
+
+### Setting up rails
+
+1. To create a rail find a counterparty who implmements this SEP in the region you wish to provide access to, and agrees to do business with you.
+1. Trade public keys with each other in order to identify and securely interoperate with each other
+1. Keep a mapping of region to home_domain, using home_domain as your intial entry point to interoperating.
+
+### User Onboarding
+
+1. User onboarding happens with the sending anchor, and is out of the scope of this spec.
+1. Sending anchor should use this to collect any KYC about the receiver it will need so it doesn't have to do it at every transaction.
+
+### Payment Flow
+
+1. Payment initiation happens in the sending anchors interface, and is also out of the scope of this spec.
+1. The sending client chooses a destination region.
+1. The sending anchor fetches the TOML file to find the SEND_SERVER API and [`/info`](#info) endpoint fields for the receiving anchor for this region.
+1. Using the fields fetched from /info, the sending anchor collects all needed information from the sending client.
+1. The sending anchor POSTs all that information to the [`/send](#send) endpoint of the receiving anchor.
+1. If information is missing, or the receiving anchor needs extra information to complete this transaction, `POST /send` will return `403 Customer Information Needed` along with the missing fields. The sending anchor should collect these and retry the entire `POST /send` call, until it succeeds.
+1. Once all needed information is provided and the receiving anchor is ready to complete this transaction, the `POST /send` call will return a `200` status along with information needed to complete the transaction over the stellar network.
+1. The sending anchor should collect the fiat from the sending client, and perform the specified stellar transaction to send the money to the receiving anchor.  This is usually a path payment, but can be done with a regular payment as well, so long as the receiving anchor gets the token they expect.
+1. Once the stellar transaction is completed, the receiving anchor should deposit the money in the receivers bank account.
+1. The sending anchor can query the status of this transaction via the [`/transaction`](#transaction) endpoint, and should communicate updates to the sending client as that progresses.
+
+
+
+## API Endpoints
+
+* [`GET /info`](#info): required
+* [`POST /send`](#send): required
+* [`GET /transactions`](#transaction-history): required
+* [`GET /transaction`](#single-historical-transaction): required
```

Comment:
Do we need the `/transactions` endpoint?  It was mostly for creating UIs for wallets and feels unnecessary here